examples/kubernetes/README.md

```diff
@@ -0,0 +1,233 @@
+# Using the Cloud SQL proxy on Kubernetes
+
+The Cloud SQL proxy is the recommended way to connect to Cloud SQL, even when
+using private IP. This is because the proxy provides strong encryption and 
+authentication using IAM, which can help keep your database secure. 
+
+## Configure your application with Secrets
+
+In Kubernetes, [Secrets][ksa-secret] are a secure way to pass configuration 
+details to your application. You can create a Secret with details such as your
+database name, user, and password which can be injected into your application as
+env vars.
+
+1. Create a secret with information needed to access your database:
+    ```shell
+    kubectl create secret generic <YOUR-DB-SECRET> \
+        --from-literal=db_user=<YOUR-DATABASE-USER> \
+        --from-literal=db_password=<YOUR-DATABASE-PASSWORD> \
+        --from-literal=db_name=<YOUR-DATABASE-NAME>
+    ```
+2. Next, configure your application's container to mount the secrets as env 
+   vars:
+    > [proxy_with_workload_identity.yaml](proxy_with_workload_identity.yaml#L12-L30)
+    ```yaml
+       env:
+        - name: DB_USER
+          valueFrom:
+            secretKeyRef:
+              name: <YOUR-DB-SECRET>
+              key: db_user
+        - name: DB_PASS
+          valueFrom:
+            secretKeyRef:
+              name: <YOUR-DB-SECRET>
+              key: db_password
+        - name: DB_NAME
+          valueFrom:
+            secretKeyRef:
+              name: <YOUR-DB-SECRET>
+              key: db_name
+    ```
+3. Finally, configure your application to use these values. In the example 
+above, the values will be in the env vars `DB_USER`, `DB_PASS`, and `DB_NAME`.
+
+## Setting up a service account
+
+The first step to running the Cloud SQL proxy in Kubernetes is creating a 
+service account to represent your application. It is recommended that you create
+a service account unique to each application, instead of using the same service
+account everywhere. This model is more secure since it allows your to limit
+permissions on a per-application basis.
+
+The service account for your application needs to meet the following criteria:
+
+1. Belong to a project with the [Cloud SQL Admin API][admin-api] enabled.
+1. [Has been granted][grant-sa] the 
+   [`Cloud SQL Client` IAM role (or equivalent)][csql-roles]
```

Comment:
Minor nit: consistent use of periods at the end of list items. Some have them and some don't here.

Reply:
Done.